data-service/src/test/java/uk/gov/laa/ccms/data/mapper/LookupMapperImplTest.java

```diff
@@ -314,9 +315,64 @@ void toRelationshipToCaseLookupDetailTest() {
         assertEquals(expectedDetail, actualDetail);
     }
 
+    @Test
```

Comment:
github won't let me add a comment on existing code.
In toRelationshipToCaseLookupDetailTest(), line 308.
        List<RelationshipToCaseLookupValueDetail> actualDetailsList =
            mapper.personRelationshipToCaseLookupValueListToRelationshipToCaseLookupValueDetailList(values);

actualDetailList is not used. Were you meaning to test two mapper methods in one test?

Reply:
Sorted